packages/react-components/react-charts-preview/stories/src/VerticalBarChart/VerticalBarChartDefault.stories.tsx

```diff
@@ -242,12 +242,12 @@ export const VCBasic = () => {
             lineLegendText={'just line'}
             lineLegendColor={'brown'}
             lineOptions={lineOptions}
-            {...(isCalloutselected && {
+            /*{...(isCalloutselected && {
```

Comment:
why disabled this

Reply:
Custom callout is not working. Needs to be fixed.

Reply:
Okay